website/docs/general/2fa.md

```diff
@@ -1,30 +0,0 @@
----
```

Comment:
Hi @navendu-pottekkat, this guide is used to remind contributors/maintainers to be more secure. When one wants to have written permission to `apache/*` projects, we have to enable 2FA.

How about adding this section to Committer Guide?

Reply:
I have moved this to the Contributing Flow guide. As every contributor should go through the contributor flow, this could be part of that flow.